test-projects/twitter-login/server.js

```diff
@@ -33,6 +33,8 @@ keystone
       });
       await keystoneApp.createItems(initialData);
     }
+
+    console.log(`Listening on port ${port}`);
```

Comment:
We don't need this message anymore, now that `createGraphQLMiddleware` and `createDevMiddleware` do a better job of logging the routes they set up.